tensorflow/compiler/mlir/lite/ir/tfl_ops.td

```diff
@@ -644,7 +644,8 @@ def TFL_ConcatenationOp : TFL_Op<"concatenation",
     ins TFL_VariadicTensorOf<
       [F32, I64, I32, I16, I8, QI8, QUI8, TFL_Uint8]>:$values,
     I32Attr:$axis,
-    TFL_AFAttr:$fused_activation_function
+    TFL_AFAttr:$fused_activation_function,
+    BoolAttr:$fixed_point_scaling
```

Comment:
Is it necessary to add a `fixed_point_scaling` field for the MLIR converter? From what I understand, the `SameOperandsAndResultsScale` trait of `TFL_ConcatenationOp` guarantees that the scaling of the inputs and output will be the same . In this case the concatenation kernel for uint8 only does a memcpy regardless of the value of the `fixed_point_scaling` in the TFLite flatbuffer.

Reply:
right. the scales will be the same and the kernel will use memory copy.